lib/src/utils/custom_number_format.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:intl/intl.dart';
+
+class CustomNumberFormat {
+  static NumberFormat customNumberFormat = NumberFormat("#,###.00", "en_US");
+  static NumberFormat customNumberFormatWithoutCommas = NumberFormat("#.00", "en_US");
+}
```

Comment:
we should turn these into helper methods, that way they can be called with parameters anywhere in the app.

Reply:
I fixed it.